wiki/People/The_Team/de.md

```diff
@@ -1,56 +1,103 @@
+| ![][flag_AU] [Zallius](https://osu.ppy.sh/users/55) | Gefährdete Art |
 
-## Aktive
+Darüber hinaus ist das [Account-Support-Team](Account_support_team) da, um bei den Dingen zu helfen, die außerhalb der Reichweite liegen.
 
-| Name | Beschreibung |
-| :-- | :-- |
-| [Ephemeral](https://osu.ppy.sh/users/102335) | Spielerunterstützer |
-| [flyte](https://osu.ppy.sh/users/3103765), [osu!next](https://next.ppy.sh/) | Chefdesigner |
-| [Loctav](https://osu.ppy.sh/users/71366) | Offizieller Turnier Organisator |
-| [nanaya](https://osu.ppy.sh/users/2387883) | Webimplementierung |
-| [RBRat3](https://osu.ppy.sh/users/307202) | osu! Zeichner, Macht coole extraordinäre Sachen |
-| [smoogipooo](https://osu.ppy.sh/users/1040328) | osu!mania Entwickler, Sucht und behebt Spielfehler |
-| [Tom94](https://osu.ppy.sh/users/1857058) | pp Entwickler |
-| [yelle](https://osu.ppy.sh/users/4916903) | Kunstwettbewerbsveranstalter, osu!store management |
-| [ztrot](https://osu.ppy.sh/users/6347) | osu!academy Professor, Medien/Charakter Design |
-| [Zallius](https://osu.ppy.sh/users/55) | Gefährdete Spezies |
-
-## Inaktive
+## Benutzergruppen
+
+Die folgenden Benutzergruppen bestehen aus Mitgliedern der osu! Community, die helfen, osu! zu erhalten. Die meisten dieser Benutzergruppen sind an den jeweiligen Forumsfarben, Chatfarben im Spiel, Profiltiteln und/oder Profilabzeichen zu erkennen.
 
 | Name | Beschreibung |
 | :-- | :-- |
-| [awp](https://osu.ppy.sh/users/2650) ([Barrel Roll Weekly](http://brw.twinkfish.com/)) | Allgemeine Haushaltsführung und Unterstützung, Chibi Illustrationen |
-| [Derekku](https://osu.ppy.sh/users/91341) | Allgemeine Haushaltsführung, Community Verwaltung |
-| [Echo](https://osu.ppy.sh/users/431) ([blog](http://blog.echo.sh/)) | osu! Entwickler Team, IRC-Integration der In-Game Chats, Seiteninstandhaltung |
-| [Intermezzo](https://osu.ppy.sh/users/136842) | osu! Entwickler Team, osz2/p2p Zuständiger |
-| Jim ([Brave New Games](http://www.bravegamer.com/)) | Originales Seitendesign |
-| [LuigiHann](https://osu.ppy.sh/users/1079) ([LuigiHann@Deviantart](http://luigihann.deviantart.com/)) | Episches Skin Design, Beiträger am "default"-Skin und an Symbol- und Zeichensets |
-| [mm201](https://osu.ppy.sh/users/30655) | osu! Entwickler Team, mm sliders |
-| [nuudles](https://osu.ppy.sh/users/21312) | osu! iPhone Entwickler |
-| [phill](https://osu.ppy.sh/users/53) | Forum Design Arbeit |
-| [Sarumaru](https://osu.ppy.sh/users/9427) ([Sarumaru@deviantART.com](http://sarumaru.deviantart.com/)) | Originales pippi Entwurf und Grafikdesign |
-| [Sinistro](https://osu.ppy.sh/users/5530) | Besprechungsprotokolle, Große FAQ Beiträger |
-| [woc2006](https://osu.ppy.sh/users/1105845) | osu! Entwickler Team, osu! osu!mania Modus Entwickler |
+| [Beatmap Nominators](Beatmap_Nominators) | Sorgen dafür, dass deine Beatmap in den "qualifiziert"-Status kommen |
```

Comment:
```suggestion
| [Beatmap Nominators](Beatmap_Nominators) | Sorgen dafür, dass deine Beatmap in den "qualifiziert"-Status kommt |
```
Either `Beatmap` and `kommt` or `Beatmaps` and `kommen`. Both is fine to me.

Reply:
oops my bad